### DISCUSSION TITLE
Crowdin strings

### DISCUSSION THREAD

**Discussion by Sontarr** [seerge/g-helper#2293] [Ideas]
Hello seerge,

I tried to translate a few of the new strings in Crowdin, but I can't find them. Will they eventually be there to translate? I'm talking f.e. about "Gamut", "Native" and "CPU Cores Configuration".

"Default" under Visual Mode and "Apply" under CPU Cores Configuration don't translate atm, but already have translations in other parts of the app.

**Comment by seerge** [comment]
@Sontarr they are just not there yet, I will add them eventually when I will be sure that this strings won't change in future

I'm also not sure if Visual Mode names should be translated (as I simply don't remember if they are translated in AC or not)

Same applies to color temperature names. I tried to be creative with Warmest / Warmer / Warm / etc naming in English, but I can imagine it may not translate that easily into other languages

**Reply by Sontarr** [reply]
@seerge I have seen the edit just yet .... and installed Armoury Crate to check. You are right: the visual modes are not translated (I would prefer "Default" to be translated though as it essentially means the modes are off). "Gamut", "Native" and "CPU Cores Configuration" are translated.

**Reply by Kikimahe** [reply]
I agree with you.
“Default” should be changed to “disabled” and translated.

There might be fewer redundant questions from people who use custom color profiles in Windows.
